dll/win32/browseui/shellbrowser.cpp

```diff
@@ -786,11 +786,13 @@ HRESULT CShellBrowser::Initialize()
 
     fToolbarProxy.Initialize(m_hWnd, clientBar);
 
-
     // create status bar
-    fStatusBar = CreateWindow(STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS |
-                    SBT_NOBORDERS | SBT_TOOLTIPS, 0, 0, 500, 20, m_hWnd, (HMENU)0xa001,
-                    _AtlBaseModule.GetModuleInstance(), 0);
+    DWORD dwStatusStyle = WS_CHILD | WS_CLIPSIBLINGS | SBARS_SIZEGRIP | SBARS_TOOLTIPS;
+    if (fStatusBarVisible)
+        dwStatusStyle |= WS_VISIBLE;
+    fStatusBar = CreateWindowExW(0, STATUSCLASSNAMEW, NULL, dwStatusStyle,
+                                 0, 0, 500, 20, m_hWnd, (HMENU)FCIDM_TB_UPFOLDER,
+                                 _AtlBaseModule.GetModuleInstance(), 0);
```

Comment:
```suggestion
    fStatusBar = ::CreateWindowExW(0, STATUSCLASSNAMEW, NULL, dwStatusStyle,
                                   0, 0, 500, 20, m_hWnd, (HMENU)FCIDM_TB_UPFOLDER,
                                   _AtlBaseModule.GetModuleInstance(), 0);
```

Reply:
I just pushed this change 😅

Reply:
For note: the (HMENU) magic value we were mentioning in a previous comment is the window control ID corresponding to the toolbar. Not sure that `FCIDM_TB_UPFOLDER` (while this has the same numerical value as seen by Doug) actually helps in that regard. (TB == toobar? why "upfolder" while we are dealing with a status-bar, etc.)

Reply:
You are right, this is incorrect. Let's make a new define for the status bar. Should I do something like ```#define IDC_STATUSBAR 0xA001```

Reply:
`IDC_STATUSBAR` (since this is not an actual menu, but a control)

Reply:
Since `0xA001` should be an identifier of a parent, shouldn't it be something else? I mean, we're creating a status bar, it couldn't be a child of `IDC_STATUSBAR`. 🤔

Reply:
> Type: HMENU
>
A handle to a menu, **or specifies a child-window identifier**, depending on the window style. For an overlapped or pop-up window, hMenu identifies the menu to be used with the window; it can be NULL if the class menu is to be used. **For a child window, hMenu specifies the child-window identifier, an integer value used by a dialog box control** to notify its parent about events. The application determines the child-window identifier; it must be unique for all child windows with the same parent window.

In other words, the same kind of resource ID one uses in dialog resource files. See existing code in ReactOS.

Reply:
Regarding `0xA001` constant, it's also used in other places of the same file, probably it needs to be updated in these places as well.

Reply:
Added a constant, now waiting for CI...